src/Zicht/Bundle/FrameworkExtraBundle/Doctrine/FunctionNode/Rand.php

```diff
@@ -13,13 +13,10 @@
 /**
  * see https://gist.github.com/919465
```

Comment:
Misschien maar gelijk `@see` van maken?

Reply:
Of deze weggooien en https://github.com/beberlei/DoctrineExtensions/blob/master/src/Query/Mysql/Rand.php gaan gebruiken